config/tailwind.config.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
     './public/*.html',
     './app/helpers/**/*.rb',
     './app/javascript/**/*.js',
-    './app/views/**/*.erb'
+    './app/views/**/*.erb',
+    '!./app/views/account/**/*.erb'
   ],
```

Comment:
For unsigned user it is styled with tailwindcss.
![image](https://github.com/user-attachments/assets/ecfe50fb-be8c-4cae-8c9a-a34f41b1afe3)

for admin it is styled with global Tailwindcss styles and Bootstrap.
![image](https://github.com/user-attachments/assets/8c4d699e-5144-49ef-91f6-f0132f7f8e07)